src/settingsmanager.hpp

```diff
@@ -49,6 +49,7 @@ class SettingsManager : public QObject
     Setting<bool> customHighlightSound;
     Setting<QString> pathHighlightSound;
     Setting<QMap<QString, QPair<bool, bool>>> highlightProperties;
+    Setting<QString> blacklistedUsers;
```

Comment:
The name `blacklistedUsers` is not the proper name as it is only used for ignoring highlights from certain users. I think `highlightUserBlacklist` or something along the lines would be a better name.